src/cow-react/modules/limitOrders/pure/Orders/OrdersTable.styled.tsx

```diff
@@ -10,6 +10,11 @@ const statusColorMap: { [key in OrderStatus]: string } = {
   [OrderStatus.EXPIRED]: '#eeaaaa',
   [OrderStatus.FULFILLED]: '#d5eab3',
   [OrderStatus.CANCELLED]: '#fcecb4',
+  // TODO: decide what color for each state
```

Comment:
If you make the colors glowing PINK, @fairlighteth will jump quicly to the recue